src/components/SearchModal/ImportToken.tsx

```diff
@@ -42,6 +44,13 @@ export function ImportToken(props: ImportProps) {
   if (intersection.size > 0) {
     return <BlockedToken onBack={onBack} onDismiss={onDismiss} blockedTokens={Array.from(intersection)} />
   }
+
+  const eventPropertiesList = tokens.map((token) => ({
```

Comment:
this feels like it could be very large, and i don't love this abstraction tbh - also we lose that this list of tokens was a complete unit by firing a bunch of events. could we do an array of addresses and one event?